pkg/controller/networkpolicy/validate.go

```diff
@@ -513,6 +518,21 @@ func (v *antreaPolicyValidator) validateTierForPolicy(tier string) (string, bool
 	return "", true
 }
 
+// validateTierForPassAction validates that rules with pass action are not created in the Baseline Tier.
+func (v *antreaPolicyValidator) validateTierForPassAction(tier string, ingress, egress []crdv1alpha1.Rule) (string, bool) {
```

Comment:
Could we check if the tier is baseline at the very beginning? If it's not, we don't need to iterate all ingress/egress rules.

Reply:
Done